### PR TITLE
A few simple changes to Hive connector

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

File: presto-hive/src/main/java/com/facebook/presto/hive/TableOfflineException.java [hunk 1]
```diff
@@ -17,26 +17,35 @@
...
+import static com.google.common.base.Strings.isNullOrEmpty;
 import static java.util.Objects.requireNonNull;
 
 public class TableOfflineException
         extends PrestoException
 {
     private final SchemaTableName tableName;
 
-    public TableOfflineException(SchemaTableName tableName)
+    public TableOfflineException(SchemaTableName tableName, boolean forPresto, String offlineMessage)
```
Comment (electrum) [inline presto-hive/src/main/java/com/facebook/presto/hive/TableOfflineException.java]: Add a test for this that verifies the exception's message is correct

File: presto-hive/src/main/java/com/facebook/presto/hive/TableOfflineException.java [hunk 2]
```diff
@@ -17,26 +17,35 @@
...
     }
 
     public SchemaTableName getTableName()
     {
         return tableName;
     }
+
+    private static String formatMessage(SchemaTableName tableName, boolean forPresto, String offlineMessage)
+    {
+        StringBuilder resultBuilder = new StringBuilder();
```
Comment (electrum) [inline presto-hive/src/main/java/com/facebook/presto/hive/TableOfflineException.java]: You can chain this
```java
StringBuilder resultBuilder = new StringBuilder()
        .append("Table '").append(tableName).append("'")
        .append(" is offline");

File: presto-hive/src/main/java/com/facebook/presto/hive/PartitionOfflineException.java [hunk 3]
```diff
@@ -51,4 +42,17 @@ public String getPartition()
     {
         return partition;
     }
+
+    private static String formatMessage(SchemaTableName tableName, String partitionName, boolean forPresto, String offlineMessage)
+    {
+        StringBuilder resultBuilder = new StringBuilder();
```
Comment (electrum) [inline presto-hive/src/main/java/com/facebook/presto/hive/PartitionOfflineException.java]:
```java
StringBuilder resultBuilder = new StringBuilder()
        .append("Table '").append(tableName).append("'")
        .append(" partition '").append(partitionName).append("'")
        .append(" is offline");

File: presto-hive/src/main/java/com/facebook/presto/hive/PartitionOfflineException.java [hunk 4]
```diff
@@ -25,21 +26,11 @@
     private final SchemaTableName tableName;
     private final String partition;
 
-    public PartitionOfflineException(SchemaTableName tableName, String partition)
+    public PartitionOfflineException(SchemaTableName tableName, String partitionName, boolean forPresto, String offlineMessage)
```
Comment (electrum) [inline presto-hive/src/main/java/com/facebook/presto/hive/PartitionOfflineException.java]: Also add a test for the exception message

File: presto-hive/src/main/java/com/facebook/presto/hive/TableOfflineException.java (hunk 1 above)
Comment (haozhun) [inline presto-hive/src/main/java/com/facebook/presto/hive/TableOfflineException.java]: Done

File: presto-hive/src/main/java/com/facebook/presto/hive/TableOfflineException.java (hunk 2 above)
Comment (haozhun) [inline presto-hive/src/main/java/com/facebook/presto/hive/TableOfflineException.java]: Done

File: presto-hive/src/main/java/com/facebook/presto/hive/PartitionOfflineException.java (hunk 4 above)
Comment (haozhun) [inline presto-hive/src/main/java/com/facebook/presto/hive/PartitionOfflineException.java]: Done

File: presto-hive/src/main/java/com/facebook/presto/hive/PartitionOfflineException.java (hunk 3 above)
Comment (haozhun) [inline presto-hive/src/main/java/com/facebook/presto/hive/PartitionOfflineException.java]: Done

**Pull request merged**